packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -503,15 +506,37 @@ class SimcoreServiceLabels(DynamicSidecarServiceLabels):
     spec will be generated before starting the service.
     """
 
-    settings: Json[SimcoreServiceSettingsLabel] = Field(
-        default_factory=dict,
-        alias="simcore.service.settings",
-        description=(
-            "Json encoded. Contains setting like environment variables and "
-            "resource constraints which are required by the service. "
-            "Should be compatible with Docker REST API."
+    settings: Annotated[
+        Json[SimcoreServiceSettingsLabel],
+        Field(
+            default_factory=dict,
+            alias="simcore.service.settings",
+            description=(
+                "Json encoded. Contains setting like environment variables and "
+                "resource constraints which are required by the service. "
+                "Should be compatible with Docker REST API."
+            ),
         ),
-    )
+    ]
+
+    assert "json_schema_extra" in SimcoreServiceSettingLabelEntry.model_config  # nosec
+    assert isinstance(
+        SimcoreServiceSettingLabelEntry.model_config["json_schema_extra"], dict
+    )  # nosec
+
+    assert "json_schema_extra" in PathMappingsLabel.model_config  # nosec
+    assert isinstance(
+        PathMappingsLabel.model_config["json_schema_extra"], dict
+    )  # nosec
+    assert isinstance(
+        PathMappingsLabel.model_config["json_schema_extra"]["examples"], list
+    )  # nosec
+
+    assert "json_schema_extra" in CallbacksMapping.model_config  # nosec
+    assert isinstance(CallbacksMapping.model_config["json_schema_extra"], dict)  # nosec
+    assert isinstance(
+        CallbacksMapping.model_config["json_schema_extra"]["examples"], list
+    )  # nosec
```

Comment:
What is the reason for these checks? I don't see what they bring

Reply:
As discussed. I also moved it to a callable as an alternaite